Core/include/Acts/Clusterization/HkClusterization.hpp

```diff
@@ -0,0 +1,75 @@
+#pragma once
+
+#include <memory>
+#include <vector>
+
+namespace Acts {
+
+typedef int Label;
+#define NO_LABEL 0
+
+// Simple wrapper around a generic cell type
+// The Cell type should have the following accessor functions defined:
+//   int get_cell_row(const Cell&)
+//   int get_cell_column(const Cell&)
+template <typename Cell>
+struct LabeledCell {
+  Cell const* ptr;
+  mutable Label lbl;
+  explicit LabeledCell(const Cell& cell) : ptr{std::addressof(cell)}, lbl{0} {}
+};
+
+// Definition of the Cell api for the LabeledCell<Cell> wrapper
+template <typename Cell>
+int get_cell_row(const LabeledCell<Cell>& cell) {
```

Comment:
Consider camelCase method naming (you use it for other functions below also)

Reply:
See my comment below; these functions will soon be removed